cloud_utils/cache/stores/lru_memory_store.py

```diff
@@ -0,0 +1,35 @@
+import collections
```

Comment:
let's remove the "store" from the file names (no need they are already under the directory)